diff --git a/c:\Users\marcd\Documents\Code\Laravel\Dakar-Terminal\application\dt.app\routes\web.php b/c:\Users\marcd\Documents\Code\Laravel\Dakar-Terminal\application\dt.app\routes\web.php
--- a/c:\Users\marcd\Documents\Code\Laravel\Dakar-Terminal\application\dt.app\routes\web.php
+++ b/c:\Users\marcd\Documents\Code\Laravel\Dakar-Terminal\application\dt.app\routes\web.php
@@ -10,2 +10,3 @@
     Route::get('dashboard', DashboardController::class)->name('dashboard');
+    Route::view('facturation/dashboard', 'facturation.dashboard')->name('facturation.dashboard');
 });
